.github/workflows/intermittent-test-check.yml

```diff
@@ -157,31 +139,28 @@ jobs:
             export OZONE_REPO_CACHED=true
           fi
 
-          test_type=${{ needs.prepare-job.outputs.test_type }}
-          args="-DexcludedGroups=unhealthy"
-          if [ "$test_type" = "integration" ]; then
-              args="$args -pl :ozone-integration-test,:mini-chaos-tests"
-          fi
+          args="-DexcludedGroups=native|slow|unhealthy"
           if [ "$TEST_METHOD" = "ALL" ]; then
-              echo "Running all tests from $TEST_CLASS"
-              hadoop-ozone/dev-support/checks/junit.sh $args -Dtest=$TEST_CLASS
+            echo "Running all tests from $TEST_CLASS"
+            set -x
+            hadoop-ozone/dev-support/checks/junit.sh $args -Dtest="$TEST_CLASS,Abstract*Test\$*"
           else
-              echo "Running test: $TEST_METHOD from $TEST_CLASS"
-              hadoop-ozone/dev-support/checks/junit.sh $args -Dtest=$TEST_CLASS#$TEST_METHOD
+            echo "Running test: $TEST_METHOD from $TEST_CLASS"
+            set -x
+            hadoop-ozone/dev-support/checks/junit.sh $args -Dtest="$TEST_CLASS#$TEST_METHOD,Abstract*Test\$*"
```

Comment:
How about AbstractRootedOzoneFileSystemTestWithFSO ? This regex is not matched right?

Reply:
Correct.  `AbstractRootedOzoneFileSystemTestWithFSO` doesn't have nested tests, so it's not an urgent problem.  But we can tweak the pattern to `Abstract*Test*$*` to handle that.

Reply:
> TestLegacyReplicationManager: no tests are run

How does this change help in running this test. It is not a concrete class inheriting any Abstract class but only has multiple Nested tests within

Reply:
Adding the `Abstract...` pattern is not needed for `TestLegacyReplicationManager` and similar classes.  For these, only items 1 and 2 are critical (removal of file find, artifact name change).

Reply:
Ok, but just wondering: in this case of `TestLegacyReplicationManager `the file should have been found and artifact name also valid right? Or is it because the right way to run nested test is to give * at the end?

Reply:
With `TestLegacyReplicationManager` the file is found and artifact name is valid, but Surefire doesn't run any tests.

`TestLegacyReplicationManager$Misc` (or similar) or `TestLegacyReplicationManager$*` select the tests in Surefire, but do not pass the file find.  Second one is also invalid artifact name.  (I don't know if the first one is valid, it didn't reach that part of the check.)

Reply:
Got it thanks for explaining. So the way to run nested tests i.e via appending * or $NestedTestName  was coming in the way of those checks.

Reply:
BTW, these are good questions, and I spent a day trying to find out what works and what doesn't.  Surefire doc is a bit lacking, especially since we must use an old version.

Reply:
Updated the patch with the more lenient `Abstract...` pattern.